### DISCUSSION TITLE
Blank Tensorboard output

### DISCUSSION THREAD

**Discussion by alikhabazian** [idiap/coqui-ai-TTS#484] [Q&A] [answered]
I have this tensor board logs that seems weird 
I share them with you please tell me it is normal or not
<img width="876" height="335" alt="Screenshot 2025-08-25 at 6 37 43 PM" src="https://github.com/user-attachments/assets/29a9f920-b4e8-45d8-af26-71c903911108" />
<img width="637" height="342" alt="Screenshot 2025-08-25 at 6 37 50 PM" src="https://github.com/user-attachments/assets/974775cf-8629-4ce2-b85f-05295255885b" />
<img width="883" height="354" alt="Screenshot 2025-08-25 at 6 38 04 PM" src="https://github.com/user-attachments/assets/7e673a27-02fa-4bb5-b443-da8050d805d7" />
<img width="582" height="343" alt="Screenshot 2025-08-25 at 6 38 11 PM" src="https://github.com/user-attachments/assets/a21f8cc5-b5e9-4c01-8fc6-6b7fca8eaadf" />
<img width="883" height="437" alt="Screenshot 2025-08-25 at 6 42 05 PM" src="https://github.com/user-attachments/assets/7cf813df-f39a-4578-be99-29698f0095d8" />

**Comment by eginhard** [comment, marked as answer]
One of your training audio files is empty or silence-only. Best to check/filter your training data first.

**Reply by alikhabazian** [reply]
thanks for your replay
I have another weird situation it seems stuck in tensor-board  on 10547 step
but it is running still
and I have log of tokenization and etc
but gpu usage decrease a lot from 13 gig to 3 gig
<img width="434" height="433" alt="image" src="https://github.com/user-attachments/assets/6c79d410-5ab4-4e23-8a45-d9da30454c3a" />
Is it logical?
<img width="1046" height="597" alt="image" src="https://github.com/user-attachments/assets/e288b245-b37b-40c1-8999-0876a9bf8781" />

**Reply by eginhard** [reply]
You'd have to check the log

**Reply by alikhabazian** [reply]
Unfortunately because of making debug logging on, it is impossible between millions of tokenizer input :(
I wait till morning it does not heal I interrupt it and continue it after

**Reply by alikhabazian** [reply]
> One of your training audio files is empty or silence-only. Best to check/filter your training data first.

Hi, model was training for 48 hours the output is this 
spectrograms seems good (which is weird it was good from scratch early train epochs)
train audio that is generated are not bad (but I can not check because I do not know what is transcription but could map to some words in Persian)
Is this correct config I use ?
```
model_args = VitsArgs(
        d_vector_file=D_VECTOR_FILES,
        use_d_vector_file=True,
        d_vector_dim=512,
        num_layers_text_encoder=10,
        speaker_encoder_model_path=SPEAKER_ENCODER_CHECKPOINT_PATH,
        speaker_encoder_config_path=SPEAKER_ENCODER_CONFIG_PATH,
        resblock_type_decoder="2",  # In the paper, we accidentally trained the YourTTS using ResNet blocks type 2, if you like you can use the ResNet blocks type 1 like the VITS model
        # Useful parameters to enable the Speaker Consistency Loss (SCL) described in the paper
        # use_speaker_encoder_as_loss=True,
        # Useful parameters to enable multilingual training
        # use_language_embedding=True,
        # embedded_language_dim=4,
    )

    # General training config, here you can change the batch size and others useful parameters
    config = VitsConfig(
        output_path=OUT_PATH,
        model_args=model_args,
        run_name=RUN_NAME,
        project_name="YourTTS",
        run_description="""
                - Original YourTTS trained using VCTK dataset
            """,
        dashboard_logger="tensorboard",
        logger_uri=None,
        audio=audio_config,
        batch_size=BATCH_SIZE,
        batch_group_size=48,
        eval_batch_size=BATCH_SIZE,
        num_loader_workers=8,
        eval_split_max_size=256,
        print_step=50,
        plot_step=100,
        log_model_step=1000,
        save_step=5000,
        save_n_checkpoints=2,
        save_checkpoints=True,
        target_loss="loss_1",
        print_eval=False,
        use_phonemes=False,
        phonemizer="espeak",
        phoneme_language="en",
        compute_input_seq_cache=True,
        add_blank=True,
        text_cleaner="multilingual_cleaners",
        characters=CharactersConfig(
            characters_class="TTS.tts.models.vits.VitsCharacters",
            pad="_",
            eos="&",
            bos="*",
            blank=None,
            characters="ءابتثجحخدذرزسشصضطظعغفقلمنهويِپچژکگیآۀةأؤإئًَُّ!(),-.:;?  ̠،…؛؟‌٪#üABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz–@+/\u200c",
            punctuations="!'(),-.:;?،؛ ",
            phonemes="",
            is_unique=True,
            is_sorted=True,
        ),
        phoneme_cache_path=None,
        precompute_num_workers=12,
        start_by_longest=True,
        datasets=DATASETS_CONFIG_LIST,
        cudnn_benchmark=False,
        max_audio_len=SAMPLE_RATE * MAX_AUDIO_LEN_IN_SECONDS,
        mixed_precision=False,
        test_sentences=[
            [
                "سلام خوبی؟",
                "MCV_6817d29767e5232282f1be1f66a1127aebb547ab822baad701f436f7df143bffb050dacb75e2f39da6f6c98b18a556af871cfa8af99aa0d717360d791e656f0c",
                None,
                "fa",
            ],
            [
                "مردمان ایران زمین",
                "MCV_739f9fa03426b388658a8ff68372dc3d9b34656cb78c3cf9be80c15b17b43c0f2cdfdaf93085ad001f3ee3d53cf005c51ef584cf5eafd8db9e1628926a74b1a1",
                None,
                "fa",
            ],
            
        ],
        # Enable the weighted sampler
        use_weighted_sampler=True,
        # Ensures that all speakers are seen in the training batch equally no matter how many samples each speaker has
        weighted_sampler_attrs={"speaker_name": 1.0},
        weighted_sampler_multipliers={},
        # It defines the Speaker Consistency Loss (SCL) α to 9 like the paper
        speaker_encoder_loss_alpha=9.0,
        resort=True,
        sorted_name='cv_ds_sorted'
    )

```
and this is my logs some loss in train are ascending which is shocking for me

<img width="1076" height="617" alt="Screenshot 2025-08-27 at 9 35 41 AM" src="https://github.com/user-attachments/assets/f1beea7b-400d-42ab-b275-06438093c31e" />
<img width="1061" height="613" alt="Screenshot 2025-08-27 at 9 36 04 AM" src="https://github.com/user-attachments/assets/3f989469-5250-4e29-963c-503a5226b0ed" />
<img width="1101" height="615" alt="Screenshot 2025-08-27 at 9 36 39 AM" src="https://github.com/user-attachments/assets/ec079b12-8523-49cd-9547-5cf9f83500b9" />
<img width="1181" height="600" alt="Screenshot 2025-08-27 at 9 38 53 AM" src="https://github.com/user-attachments/assets/9aae8a03-e5a6-4ab6-920f-88bc1dab4dd5" />
<img width="1189" height="479" alt="Screenshot 2025-08-27 at 9 39 04 AM" src="https://github.com/user-attachments/assets/2dfb9dd9-979c-48ed-86d2-586a41ef40fe" />
<img width="1181" height="623" alt="Screenshot 2025-08-27 at 9 39 23 AM" src="https://github.com/user-attachments/assets/75121e7b-2087-49e0-a68c-efa78be99f25" />
<img width="1191" height="327" alt="Screenshot 2025-08-27 at 9 39 32 AM" src="https://github.com/user-attachments/assets/d5062ac6-6ee9-4987-963c-2877a12c6de0" />
<img width="1200" height="340" alt="Screenshot 2025-08-27 at 9 40 50 AM" src="https://github.com/user-attachments/assets/f0d54682-8dd1-44a4-b45c-21fb36a12c53" />

and I did not make ture use_speaker_embedding in config does it use speaker embedding too
